pandas/core/window/rolling.py

```diff
@@ -612,6 +631,126 @@ def aggregate(self, func, *args, **kwargs):
     """
     )
 
+    _shared_docs["var"] = dedent(
+        """
+    Calculate unbiased %(name)s variance.
```

Comment:
Can you add a `versionadded` for Window here? NDFrame.pipe and GroupBy.pipe should have an example.